pkg/config/user_config.go

```diff
@@ -164,6 +164,9 @@ type KeybindingUniversalConfig struct {
 	SubmitEditorText             string `yaml:"submitEditorText"`
 	AppendNewline                string `yaml:"appendNewline"`
 	ExtrasMenu                   string `yaml:"extrasMenu"`
+	ToggleWhitespaceInDiffView   string `yaml:"toggleWhitespaceInDiffView"`
+	IgnoringWhitespaceInDiffView string `yaml:"ignoringWhitespaceInDiffView"`
```

Comment:
I believe these two additional fields are here in error: we only need `IgnoringWhitespaceInDiffView` and `ShowingWhitespaceInDiffView` for the sake of displaying text to the user: we don't need these involved in keybindings

Reply:
yeah, this was an error on my part, I think when I was trying to get in the translations strings for the feedback you requested; removed.